### PR TITLE
feat: (Platform) introduce Feed Input component

### PR DESCRIPTION
**Pull request by genereg1**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/2847

#### Please provide a brief summary of this pull request.
Introduce Platform Feed Input component

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/platform/src/lib/components/feed-input/feed-input.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="fd-feed-input" [class.is-disabled]="disabled">
+    <fd-avatar *ngIf="avatarShow"
+        class="fd-feed-input__thumb"
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/feed-input/feed-input.component.html:3]: I wrote in another PR that in my opinion this should be a directive.

Comment (genereg1) [reply libs/platform/src/lib/components/feed-input/feed-input.component.html:3]: I think the platform implementation may have not a directive. In the case of the сore implementation agree with you. The directive would help avoid to bind external class to be easier to usage component. Here the class just underhood the component.

**Comment by stefanoScalzo** [comment]
The submit button is constantly disabled

**Comment by genereg1** [comment]
Hi, @stefanoScalzo The submit button is disabled when the text area hasn't value

**Review by Betrozov — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-feed-input/platform-feed-input-docs.component.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<h2>Standard</h2>
```
Comment (salarenko) [inline apps/docs/src/app/platform/component-docs/platform-feed-input/platform-feed-input-docs.component.html:1]: You should use `fd-docs-section-title` component for documentation header.

Comment (genereg1) [reply apps/docs/src/app/platform/component-docs/platform-feed-input/platform-feed-input-docs.component.html]: @salarenko Changed

**Review by salarenko — APPROVED** [review]

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Pull request merged**